### DISCUSSION TITLE
Is there any doc about how to unpack the watchface bin file？

### DISCUSSION THREAD

**Discussion by huada-huang** [ooflet/Mi-Create#30] [Questions-EN]
Hi,I am trying the code on Mac and it seems work by without binary file for compile or pack/unpack.I do found compile or unpack binary file for windows.I am figuring if it is possible to pack/unpack the watchface bin file for Miband9.I know that it is something like Transcopy disk image format which is not quite like Miband7.I can directly unpack the Miband7 bin file with zip app.Tanks.

**Comment by ooflet** [comment]
You can use Wine on macos to run the compile and unpack EXEs. Make sure you install wine mono when asked to, because it is necessary for the compiler and unpacker (they are made using C# + .net)

Also how is the macos compatibility? I'm looking to support that platform 😄
